Functions.Templates/Templates/SqlInputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async function (context, results) {
+    if (!!results && results.length > 0) {
+        context.log('Query results: ', JSON.stringify(results));
+    } else {
+        context.log("No results!");
+    }
+    return {
+        status: 201,
```

Comment:
201 is created, which is incorrect for an input binding. It should just be 200 "OK".